sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/PropertiesUtils.java

```diff
@@ -234,6 +238,40 @@ public static Properties getModuleProperty(String url) {
 		return properties;
 	}
 
+	/**
+	 * Retrieves the OpenMRS SDK properties from sdk.properties file
+	 *
+	 * @return A {@link Properties} object containing the OpenMRS SDK properties, or an empty {@link Properties} object
+	 *         if the SDK properties file does not exist.
+	 * @throws MojoExecutionException If there is an error during the execution of this method, such as file I/O errors.
+	 */
+	public static Properties getSdkProperties() throws MojoExecutionException {
+		File sdkFile = Server.getServersPath().resolve(SDKConstants.OPENMRS_SDK_PROPERTIES).toFile();
+		if (sdkFile.exists()){
+			return loadPropertiesFromFile(sdkFile);
+		}
+		return new Properties();
+	}
+
+	/**
+	 * Saves the provided properties to a file.
+	 *
+	 * @param properties The {@link Properties} to be saved to the file.
+	 * @param file       The {@link File} where the properties will be saved.
+	 * @throws MojoExecutionException If an {@link IOException} occurs during the file writing process,
+	 *                               or if there is an issue with the provided properties or file.
+	 */
+	public static void savePropertiesChangesToFile(Properties properties, File file)
+			throws MojoExecutionException {
+		try (OutputStream fos = new FileOutputStream(file)) {
+			properties.store(fos, SDK_PROPERTIES_FILE + ":");
```

Comment:
What is the use of the full colon at the end of this comment?

Reply:
I removed the colon.